datafusion/core/src/physical_plan/planner.rs

```diff
@@ -1157,21 +1157,11 @@ impl DefaultPhysicalPlanner {
                         "Unsupported logical plan: Dml".to_string(),
                     ))
                 }
-                LogicalPlan::TransactionStart(_) => {
+                LogicalPlan::Statement(statement) => {
```

Comment:
this is the key difference -- collapse several LogicalPlan variants into one that all have the same behavior